assignments/README.md

```diff
@@ -7,9 +7,9 @@ This assignment uses life expectancy in Europe grouped by Country (or other, lik
 ## Installing
 
 1. Clone this repo.
-2. Create a virtual environment with `python -m venv .venv`
-3. Activate the virtual environment with `source .venv/bin/activate` or `.venv\Scripts\activate` on Windows
-4. Install its dependencies on editable mode with `pip install -e .[dev]`.
+2. Create a virtual environment with `python -m venv .venv`. If you are using conda, you can create a virtual environment with `conda create --name foundations`.
```

Comment:
```suggestion
2. Create a virtual environment with `python -m venv .venv`.
  > **Note for Anaconda users**: We would prefer if you exit the automatic conda environments and tried using the steps above, as they are the canonical Python way of creating virtual environments. However, we realize working that way might be tricky for Anaconda users since Anaconda usually changes the configurations of your machine. If you are having too many problems getting started, then feel free to use the "conda way" of handling environments.  I.e.: create a virtual environment with `conda create --name foundations`.
```